scripts/nf-core

```diff
@@ -229,7 +229,7 @@ def create(name, description, author, new_version, no_git, force, outdir):
 @click.option(
     '--release',
     is_flag = True,
-    default = os.environ.get('TRAVIS_BRANCH') == 'master' and os.environ.get('TRAVIS_REPO_SLUG', '').startswith('nf-core/') and not os.environ.get('TRAVIS_REPO_SLUG', '') == 'nf-core/tools',
+    default = os.path.basename(os.path.dirname(os.environ.get('GITHUB_REF'))) == 'master' and os.environ.get('GITHUB_REPOSITORY', '').startswith('nf-core/') and not os.environ.get('GITHUB_REPOSITORY', '') == 'nf-core/tools',
```

Comment:
```suggestion
    default = os.path.basename(os.path.dirname(os.environ.get('GITHUB_REF','').strip(' \'"'))) == 'master' and os.environ.get('GITHUB_REPOSITORY', '').startswith('nf-core/') and not os.environ.get('GITHUB_REPOSITORY', '') == 'nf-core/tools',
```

Technically we should _probably_ check that `GITHUB_REF` is pointing to a branch and not a tag here, but that would only be an issue if someone tries to make a release tag called `master`. Which, let's face it, is never going to happen. Also then it would be a release, so we would also want to run these release-tests anyway.

Reply:
See https://github.com/nf-core/tools/pull/502/commits/277067f4c40f3a4332ca80af8f052d594778c587